py/aou_workbench_client/cohorts.py

```diff
@@ -1,48 +1,40 @@
 from aou_workbench_client.auth import get_authenticated_swagger_client
 from aou_workbench_client.config import all_of_us_config
 from aou_workbench_client.swagger_client.apis.cohorts_api import CohortsApi
-from copy import deepcopy
 
-def materialize_cohort_page(request, debug=False):
-    """Returns a MaterializeCohortResponse representing a page of results
-    from materializing a cohort in the workspace containing this notebook, based
-    on the provided MateralizeCohortRequest."""
+def get_data_table_query(data_table_specification, debug=False):
+    """Returns a CdrQuery that can be used to retrieve data from BigQuery for the
+    specified data table specification.
+
+    :param data_table_specification: the DataTableSpecification to get the query for
+    :param debug: true if debug request and response information should be
+    displayed; defaults to false.
+    :return a query with the SQL and configuration to use when retrieving data
+    from BigQuery. If SQL is not present in the response, the specification
+    matches no participants and there is no need to run a query against BigQuery;
+    the resulting data should be empty.
+    """
     client = get_authenticated_swagger_client(debug=debug)
     cohorts_api = CohortsApi(api_client=client)
-    return cohorts_api.materialize_cohort(all_of_us_config.workspace_namespace,
-                                          all_of_us_config.workspace_id,
-                                          request=request)
+    return cohorts_api.get_data_table_query(all_of_us_config.workspace_namespace,
+                                            all_of_us_config.workspace_id,
+                                            request=data_table_specification)
 
-def materialize_cohort(request, max_results=None, debug=False):
```

Comment:
Is it not possible to continue providing this with a deprecation notice? Or is it incompatible? I don't think it's worth doing significant work to have backwards compatibility, but the transition here might be a little bit rough, e.g. for old demo notebooks.

Reply:
I checked in with @fonnesbeck  on #workbench-beta about this. He thought it'd be OK to make a breaking change to the client libs. Note that we're maintaining server backwards compatibilty (at least for now), so if you don't want your notebook to break, you can simply not upgrade your pyclient library. (We're not automatically updating pyclient libs yet, right?)

Once we actually have launched and have notebooks out in the wild, we will obviously need to be way more careful about maintaining backwards compatibility in client libraries, too.

Reply:
We are automatically upgrading client libraries when we recreate the cluster, i.e. on a 1w-2w interval depending on when they last signed in.

As long as it won't disrupt user testing it's probably fine (I believe they are creating new workspaces in user testing, but worth confirming). Note that we will also have cases where people are looking at the same notebook from different clusters which have different client library versions.

Reply:
OK. We always have the option of not changing the client library that gets installed by workbench right away, if we want to keep the existing stuff running for longer. I'll check to make sure this doesn't mess with user testing.